presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -84,6 +84,13 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+Tables with Unsupported Columns
+-------------------------------
+
+When you query a PostgreSQL table via the Presto connector, and the table either has no supported columns or contains
```

Comment:
```suggestion
If you query a PostgreSQL table with the Presto connector, and the table either has no supported columns or contains
```
Avoid Latin abbreviations. See the [GitLab documentation style guide recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via). I made one suggestion here. See the entry for via in the link for other phrasing you could use.